R/wwinference.R

```diff
@@ -92,17 +106,17 @@ wwinference <- function(ww_data,
   if (generate_initial_values) {
     init_lists <- c()
     for (i in 1:mcmc_options$n_chains) {
-      init_lists[[i]] <- get_inits(stan_data, params)
+      init_lists[[i]] <- get_inits_for_one_chain(stan_args, params)
     }
   }
 
 
   fit_model <- function(compiled_model,
-                        standata,
+                        stan_args,
```

Comment:
I made the choise to switch to `stan_args` in part because I want to return to the user both the "data" passed to stan (these are dataframes in the `input_data` list in a subsequent PR #59 ), and the `stan_args` which is a nested list with only vectors, matrices, scalars, integers etc. I think the reason I prefer args is to distinguish between the many things we are calling `data` which are almost always in the form of a dataframe/tibble.

Reply:
I think args is general enough as arguments. I think of configuration arguments as specifications

Reply:
Counterpoint: this list is specifying the thing called the `data` block in a Stan model and is being passed as the `data` named argument to `sample()`. Calling it something with `data` in it makes that clearer. Can we compromise on `stan_data_list` as suggested below? 

If you want to distinguish it from processed but not `sample()`-`data`-argument-formatted data tables, I think calling the others `input_ww_data` or `input_count_data` versus a `stan_data_list` suffices.